sql-gremlin/README.asciidoc

```diff
@@ -66,11 +68,13 @@ Supported operators are listed below. Arithmetic, string, conditional and date o
 
 ==== Comparison Operators
 * `value1 <op> value2` where `op` is one of : `=`, `<>`, `<`, `>`, `<=` or `>=`
+* `NOT` in conjunction with a comparison operator is not currently supported
 
 ==== Logical Operators
 * `boolean1` OR `boolean2`
 * `boolean1` AND `boolean2`
-* NOT `boolean2`
+* NOT `boolean2` is supported only when used with a boolean valued column in `WHERE`/`HAVING` filters
+* NOT `boolean2` is supported with expressions and boolean values in `SELECT` clause
```

Comment:
I don't think we should be calling it `boolean2` here.

I also think this wording can be improved to be more clear that we're talking about expressions and column type values but this is a NIT.